### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by lucperkins** [liip/TheA11yMachine#134]
Could someone please push a new release of this project? Version 0.9.3 is simply broken and will not install.

**Comment by chademkarp** [comment]
I agree with @lucperkins, I would love to try this out, but it appears to be in a broken state at the moment. Please and thank you. Maybe more people could be given access to help update the system, merge pull requests, etc. ?

**Comment by julianpinedayyz** [comment]
Same page here. I would love to see an update. Thanks!
